app-web/src/components/Home/ResourcePreview.js

```diff
@@ -26,6 +26,7 @@ import { Container, LinkContainer } from './index';
 import Card from '../Cards/Card/Card';
 import Pill from '../UI/Pill';
 import { RESOURCE_TYPES } from '../../constants/ui';
+import { getSearchResultLable } from '../../utils/helpers';
```

Comment:
mis-spelling here - should be "label" not "lable"

Reply:
Thanks, I will fix this in my next pr.